dotnet/src/Microsoft.AutoGen/Agents/HostBuilderExtensions.cs

```diff
@@ -28,7 +29,11 @@ public static AgentApplicationBuilder AddAgentWorker(this IHostApplicationBuilde
                     EnableMultipleHttp2Connections = true,
                     KeepAlivePingDelay = TimeSpan.FromSeconds(20),
                     KeepAlivePingTimeout = TimeSpan.FromSeconds(10),
-                    KeepAlivePingPolicy = HttpKeepAlivePingPolicy.WithActiveRequests
+                    KeepAlivePingPolicy = HttpKeepAlivePingPolicy.WithActiveRequests/*,
+                    SslOptions = new SslClientAuthenticationOptions
```

Comment:
Seems like this is a workaround that i mentioned, maybe this got accidently checked in. should we remove this ?

Reply:
oh yes.